gateway/src/proxy.rs

```diff
@@ -139,7 +139,7 @@ impl UserProxy {
             .await?;
 
         // Record current project for tracing purposes
-        span.record("project", &project_name.to_string());
+        span.record("shuttle.project.name", &project_name.to_string());
```

Comment:
Remember to also update the `field::Empty` name in the span / instrument

Reply:
Done, I missed on in task.rs as well, fixed now.